meta-mender-octavo-osd32mp/conf/machine/include/osd32mp1.inc

```diff
@@ -0,0 +1,45 @@
+include conf/machine/include/st-machine-common-stm32mp.inc
+include conf/machine/include/st-machine-providers-stm32mp.inc
+
+# Define specific familly common machine name
+MACHINEOVERRIDES .= ":stm32mp1common:stm32mp15common"
+
+# =========================================================================
+# Chip architecture
+# =========================================================================
+DEFAULTTUNE = "cortexa7thf-neon-vfpv4"
+include conf/machine/include/arm/armv7a/tune-cortexa7.inc
+
+# =========================================================================
+# boot scheme
+# =========================================================================
+BOOTSCHEME_LABELS += "trusted"
+#BOOTSCHEME_LABELS += "optee"
+
+OSD32MP1_DEVICETREE = "stm32mp157c-osd32mp1-red"
+
+STM32MP_DEVICETREE:append = " ${OSD32MP1_DEVICETREE}"
+
+# =========================================================================
+# Machine features
+# =========================================================================
+#MACHINE_FEATURES += "splashscreen"
```

Comment:
very nitpicky - feel free to ignore, just as above: maybe add a small explanation of the `MACHINE_FEATURES` and why these specific ones are selected.

Reply:
Added a small note about why some features are disabled.